Source/Scene/Cesium3DTileBatchTable.js

```diff
@@ -1202,6 +1202,15 @@ define([
         }
     }
 
+    function updateDerivedCommandsWireframe(derivedCommands, command) {
+        for (var name in derivedCommands) {
+            if (derivedCommands.hasOwnProperty(name)) {
+                var derivedCommand = derivedCommands[name];
+                derivedCommand.primitiveType = command.primitiveType;
+            }
+        }
+    }
+
```

Comment:
This can be combined with `updateDerivedCommandsShadows` and the function renamed to just `updateDerivedCommands`.